tests/src/raw_pdu_test.cpp

```diff
@@ -0,0 +1,21 @@
+#include <gtest/gtest.h>
+#include <tins/rawpdu.h>
+
+using namespace Tins;
+
+class RawPDUTest : public testing::Test {
+public:
+};
+
+TEST_F(RawPDUTest, ConstructFromPayloadType) {
+RawPDU::payload_type payload;
```

Comment:
Can you indent this whole section 4 spaces to the right?

Reply:
Of course - in fact, that was shocking that I'd not. Sorry!